services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitFormNew.tsx

```diff
@@ -241,7 +237,10 @@ export const ExplosivesPermitFormNew: FC<ExplosivesPermitFormProps &
           <Button
             disabled={isAmend}
             type="primary"
-            onClick={() => handleOpenAddExplosivesPermitModal()}
+            onClick={(e) => {
+              e.preventDefault();
+              handleOpenAddExplosivesPermitModal();
```

Comment:
I happen to have this function in front of me- `handleOpenAddExplosivesPermitModal` already does contain event.preventDefault() within the function body. 🤔 
Does it need to be the first line?

Reply:
oh wait, the event isn't being passed in, so the function can't prevent default. Can you pass it in just like `onClick={handleExplosivesPermitModal}` so it automatically receives it or instead pass e to the function?

Reply:
@taraepp, That's super neat, updated accordingly.